### ISSUE TITLE
Test for correctness: learning age and sex fertility probabilities [SYNTHETIC DATA]

### ISSUE THREAD

**Issue by varunsatish** [varunsatish/llama-recipes-fertility#27]
**Testing process**
Training data: age x sex pairs with fertility outcome from fake data. Data is generated by taking publicly available birth rates.

Fine-tune model, then conduct inference to obtain predicted probabilities for each age x sex pair (there should be ~ 90).

Compare predicted probabilities to actual probabilities calculated from real data.

**Output**
Create plots, separately for different amounts of training data:

y axis: probability
x axis: age
points: predicted vs real
facet: sex
